config/samples/nemo/latest/apps_v1alpha1_nemocustomizer.yaml

```diff
@@ -28,310 +28,29 @@ spec:
-      # @default -- This object has the following default values for the Phi-4.
-      microsoft/phi-4:
-        # -- Whether to enable the model.
-        enabled: false
-        # -- NGC model URI for Phi-4 model.
-        model_uri: ngc://nvidia/nemo/phi-4:1.0
-        # -- Path where model files are stored.
-        model_path: phi-4
-        # -- Training options for different fine-tuning methods.
-        training_options:
-          - training_type: sft
-            finetuning_type: lora
-            num_gpus: 1
-            num_nodes: 1
-        # -- Micro batch size for training.
-        micro_batch_size: 1
-        # -- Maximum sequence length for input tokens.
-        max_seq_length: 4096
-        # -- Number of model parameters.
-        num_parameters: 14659507200
-        # -- Model precision format.
-        precision: bf16
-        # -- Template for formatting prompts.
-        prompt_template: "{prompt} {completion}"
-
-      # -- Llama 3.3 70B Instruct model configuration.
-      # @default -- This object has the following default values for the Llama 3.3 70B Instruct model.
-      meta/llama-3.3-70b-instruct:
-        # -- Whether to enable the model.
-        enabled: false
-        # -- NGC model URI for Llama 3.3 70B Instruct model.
-        model_uri: ngc://nvidia/nemo/llama-3_3-70b-instruct:2.0
-        # -- Path where model files are stored.
-        model_path: llama-3_3-70b-instruct_0_0_1
-        # -- Training options for different fine-tuning methods.
-        training_options:
-          - training_type: sft
-            finetuning_type: lora
-            num_gpus: 4
-            num_nodes: 1
-            tensor_parallel_size: 4
-        # -- Micro batch size for training.
-        micro_batch_size: 1
-        # -- Maximum sequence length for input tokens.
-        max_seq_length: 4096
-        # -- Number of model parameters.
-        num_parameters: 70000000000
-        # -- Model precision format.
-        precision: bf16-mixed
-        # -- Template for formatting prompts.
-        prompt_template: "{prompt} {completion}"
+  entitStoreIrl: http://nemoentitystore-sample.nemo.svc.cluster.local:8000
+  nemoDatastoreUrl: http://nemodatastore-sample.nemo.svc.cluster.local:8000
+  mlflowTrackingUrl: http://mlflow-tracking.nemo.svc.cluster.local:80
+  nemoDatastoreTools:
+    image: nvcr.io/nvidia/nemo-microservices/nds-v2-huggingface-cli:25.04
+    imagePullSecrets: ngc-secret
+  modelDownloadJobs:
+    image: "nvcr.io/nvidia/nemo-microservices/customizer-api:25.04"
+    imagePullPolicy: "IfNotPresent"
+    imagePullSecrets:
+      - name: ngc-secret
+    ngcAPISecret: ngc-api-secret
+    ngcAPISecretKey: "NGC_API_KEY"
+    securityContext:
+      fsGroup: 1000
+      runAsNonRoot: true
+      runAsUser: 1000
+      runAsGroup: 1000
+    # -- Time to live in seconds after the job finishes.
+    ttlSecondsAfterFinished: 600
+    # -- Interval in seconds to poll for model download status.
+    pollIntervalSeconds: 15
```

Comment:
I think we can also uplevel this.

Reply:
These are the defaults @mkhaas, users can tune them.